test/metabase/api/card_test.clj

```diff
@@ -2378,3 +2378,30 @@
           (is (not (contains? (task.persist-refresh/job-info-for-individual-refresh)
                               (u/the-id parchived)))
               "Scheduled refresh of archived model"))))))
+
+(deftest template-tags-model-validation
```

Comment:
Could you please move this to under

https://github.com/metabase/metabase/blob/master/test/metabase/api/card_test.clj#L323-L325

or

https://github.com/metabase/metabase/blob/master/test/metabase/api/card_test.clj#L762-L764 

This file is approach 3k lines and we should keep the tests sectioned.

Reply:
I split this deftest into both sections.

https://github.com/metabase/metabase/pull/24902/commits/5be754da9da32b5794af5554b739a21ae31f1894#diff-79e8112ab508ce3896120468bf6815976c3fc3ad17a00ea48f26918dbb64ed4fL385